src/odb/src/3dblox/3dblox.cpp

```diff
@@ -693,22 +692,28 @@ dbInst* ThreeDBlox::createBump(const BumpMapEntry& entry, dbBlock* block)
   inst->setOrigin(entry.x * dbus, entry.y * dbus);
   inst->setPlacementStatus(dbPlacementStatus::FIRM);
 
-  // Net entry doesn't make sense
-  if (entry.net_name != "-") {
-    dbNet* net = block->findNet(entry.net_name.c_str());
-    if (net == nullptr) {
+  dbNet* net = nullptr;
+  dbBTerm* term = nullptr;
+
+  // Find bterm
+  if (entry.port_name != "-") {
+    term = block->findBTerm(entry.port_name.c_str());
+    if (term == nullptr) {
       logger_->error(utl::ODB,
                      539,
-                     "3DBV Parser Error: Bump net {} not found",
-                     entry.net_name);
+                     "3DBV Parser Error: Bump port {} not found",
+                     entry.port_name);
     }
+    net = term->getNet();
+  }
```

Comment:
@gadfort Why not validate both independently? If _port_name_ is unspecified for any reason, we can still rely on _net_name_ to resolve the net. Otherwise, _net_name_ is effectively redundant, as it is neither needed nor used by the bump map.
```
if (entry.port_name != "-") {
  // set the bterm and the net.
  ...
}
if (entry.net_name != "-") {
  dbNet* cur_net = block->findNet(entry.net_name.c_str());
  if (cur_net == nullptr) {
    // error for net not found
    ...
  }
  if (net != nullptr && cur_net != net) {
    // error for net mismatch with the bump's port net
    ...
  }
  net = cur_net;
}
```